src/main/java/sirius/web/http/WebContext.java

```diff
@@ -1436,7 +1436,18 @@ private long determineSessionCookieTTL() {
      * @return the two-letter code of the accepted language of the user agent or <tt>null</tt> if no valid accept
      * language was found
      */
+    // todo: deprecate
     public Optional<String> getLang() {
+        return getLanguage();
+    }
+
+    /**
+     * Returns the accepted language of the client as two-letter language code.
+     *
+     * @return the two-letter code of the accepted language of the user agent or <tt>null</tt> if no valid accept
+     * language was found
+     */
+    public Optional<String> getLanguage() {
```

Comment:
Man könnte ihn in dem Zuge auch fetchLanguage nennen

Reply:
Erledigt mit weiteren Umbenennungen